src/test/resources/scenario/test_stub.js

```diff
@@ -0,0 +1,31 @@
+Load
+    .config(
+        {
+            "item" : {
+                "data" : {
+                    "size" : ITEM_DATA_SIZE
+                }
+            },
+            "load": {
+                "op": {
+                    "limit": {
+                        "count": TEST_STEP_LIMIT_COUNT
+                    }
+                }
+            },
+            "output": {
```

Comment:
please remove the `output` section here as far as we will need the output log files for the validation

Reply:
Complete